Assets/_BForBoss/Scripts/SpeedometerBehaviour.cs

```diff
@@ -0,0 +1,55 @@
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace BForBoss
+{
+    public interface ICharacterSpeed
+    {
+        float Speed { get; }
+    }
+    
+    public class SpeedometerBehaviour : MonoBehaviour
+    {
+        [SerializeField] private Image _meter = null;
+        [SerializeField] private TMP_Text _speedLabel = null;
+        
+        private ICharacterSpeed _characterSpeed = null;
+        private float _maxSpeed = 50f;
```

Comment:
Should this have the `[SerializedField]` attribute on it so it's easier to edit?

Reply:
yeah probably. this is sort of a placeholder.